source/nodejs/adaptivecards-designer/src/dialog.ts

```diff
@@ -66,9 +66,11 @@ export abstract class Dialog {
             dialogFrameElement.onclick = (e) => {
                 // disable click bubbling from the frame element -- otherwise it'll get to the overlay, closing the
                 // dialog unexpectedly
+                if(this.preventDefaultAction) {
```

Comment:
I don't think this is necessary, since `e.cancelBubble = true` is there precisely to avoid that the dialog closes.

Reply:
We are having input type file button on Image Selection popup to browse the files. Due to this cancelBubble set to true, its preventing the file selection pop up. Added a flag to enable bubble action on Image selection popup.